src/fileinfo/file_information/file_information.cpp

```diff
@@ -1628,460 +1655,452 @@ bool FileInformation::hasResourceTableRecords() const
- * @return Title of selected certificate subject
- */
-std::string FileInformation::getCertificateSubjectTitle(std::size_t index) const
-{
-	return certificateTable.getCertificateSubjectTitle(index);
-}
-
-/**
- * Get certificate subject surname
- * @param index Index of selected certificate (indexed from 0)
- * @return Surname of selected certificate subject
- */
-std::string FileInformation::getCertificateSubjectSurname(std::size_t index) const
-{
-	return certificateTable.getCertificateSubjectSurname(index);
-}
-
-/**
- * Get certificate subject given name
- * @param index Index of selected certificate (indexed from 0)
- * @return Given name of selected certificate subject
- */
-std::string FileInformation::getCertificateSubjectGivenName(std::size_t index) const
-{
-	return certificateTable.getCertificateSubjectGivenName(index);
-}
-
-/**
- * Get certificate subject initials
- * @param index Index of selected certificate (indexed from 0)
- * @return Initials of selected certificate subject
- */
-std::string FileInformation::getCertificateSubjectInitials(std::size_t index) const
-{
-	return certificateTable.getCertificateSubjectInitials(index);
-}
-
-/**
- * Get certificate subject pseudonym
- * @param index Index of selected certificate (indexed from 0)
- * @return Pseudonym of selected certificate subject
- */
-std::string FileInformation::getCertificateSubjectPseudonym(std::size_t index) const
-{
-	return certificateTable.getCertificateSubjectPseudonym(index);
-}
-
-/**
- * Get certificate subject generation qualifier
- * @param index Index of selected certificate (indexed from 0)
- * @return Generation qualifier of selected certificate subject
- */
-std::string FileInformation::getCertificateSubjectGenerationQualifier(std::size_t index) const
-{
-	return certificateTable.getCertificateSubjectGenerationQualifier(index);
-}
-
-/**
- * Get certificate subject email address
- * @param index Index of selected certificate (indexed from 0)
- * @return Email address of selected certificate subject
- */
-std::string FileInformation::getCertificateSubjectEmailAddress(std::size_t index) const
-{
-	return certificateTable.getCertificateSubjectEmailAddress(index);
-}
-
-/**
- * Find out if there are any records in certificate table
- * @return @c true if certificate table is not empty, @c false otherwise
- */
-bool FileInformation::hasCertificateTableRecords() const
-{
-	return certificateTable.hasRecords();
-}
-
-/**
- * Find out if there is signer certificate
- * @return @c true if there is signer certificate, @c false otherwise
- */
-bool FileInformation::hasCertificateTableSignerCertificate() const
-{
-	return certificateTable.hasSignerCertificate();
-}
-
-/**
- * Find out if there is counter-signer certificate
- * @return @c true if there is counter-signer certificate, @c false otherwise
- */
-bool FileInformation::hasCertificateTableCounterSignerCertificate() const
-{
-	return certificateTable.hasCounterSignerCertificate();
-}
+// /**
```

Comment:
So much dead code!
Was it moved somewhere? Is it not needed anymore?
Unless it is super useful to have this here, remove it.